monorepo/website/src/pages/docs/how-to/edit-group.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: ../../../layouts/DocLayout.astro
+title: "Editing group information"
+order: 22
+---
+import CautionBox from '../../../components/asides/CautionBox.astro';
+
+Any member of a group can edit group information, which means that you can edit group information for all the groups that you are a member of.
+
+1. Log into your account and enter your account page by clicking 'My account' in the top-right corner of the webpage
+2. Under 'My groups' click the group that you want to edit
+3. In the group detail view, click the 'Edit group' button in the top right corner
+4. Edit any property of the group you want to change and click 'Update group' to save your changes
+
+<CautionBox>
+If you change the name of a group the center name in corresponding ENA submissions will also change.
```

Comment:
```suggestion
If you change the name of a group, the [`center_name`](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/programmatic.html#identifying-submitters) of all future, but not past, ENA submissions will change accordingly.
```

Reply:
I don't know off the top of my head - we might use both in different places - @anna-parker maybe you know it immediately?

Reply:
Ah sorry - so currently we do not (yet) do updates in ENA submission, center name is used in the project and assembly. For new assemblies we will use the new center name but projects will (for now) keep the center name that was current when they were created. 

We should add updating in future - might not be trivial though